controllers/reviews/addReview/swagger.js

```diff
@@ -0,0 +1,111 @@
+export const addReview = {
+  components: {
+    schemas: {
+      Review: {
+        type: 'object',
+        required: ['id', 'owner', 'dish', 'rating', 'review'],
+        properties: {
+          id: {
+            type: 'string',
+            description: 'The auto-generated id of the review',
+          },
+          owner: {
+            type: 'string',
+            description: 'The id of the review owner',
+          },
+          dish: {
+            type: 'string',
+            description: 'The id of the reviewed dish',
+          },
+          rating: {
+            type: 'integer',
+            format: 'int32',
+            description: 'The rating given to the dish (integer value)',
+          },
+          review: {
+            type: 'string',
+            description: 'The review text',
+          },
+        },
+        example: {
+          id: '655751742149531565aa9ec8',
+          owner: '6555c5df8104558256584cb7',
+          dish: '654fe4de18f1f5f14f2e1cc1',
+          rating: 3,
+          review: 'Can be better',
+        },
+      },
+      BearerAuth: {
+        type: 'http',
+        scheme: 'bearer',
+        description: 'Bearer token for authentication',
+      },
+    },
+  },
+  tags: [
+    {
+      name: 'Reviews',
+      description: 'The reviews managing API',
+    },
+  ],
+  paths: {
+    '/reviews': {
+      post: {
+        summary: 'Create a new review',
+        tags: ['Reviews'],
+        security: [
+          {
+            BearerAuth: [], // Use the BearerAuth security scheme for authentication
+          },
+        ],
+        parameters: [
+          {
+            in: 'header',
+            name: 'Authorization',
+            required: true,
+            schema: {
+              type: 'string',
+            },
+            description: 'Bearer token for authentication',
+          },
+        ],
+        requestBody: {
+          required: true,
+          content: {
+            'application/json': {
+              schema: {
+                $ref: '#/components/schemas/Review',
+              },
+            },
+          },
+          description: 'Bearer token for authentication',
```

Comment:
copy-pasted, I think

Reply:
I have wrote this code in YAML in my own local repo. But for our project I had to transformed it to js. 
I see, what I can already change.
I will make some changes in next commit.
If you have any specific suggestions, I'd be grateful for your guidance.

Reply:
I meant `description` text specifically, this is a schema for `body`, but it says about bearer still